src/components/habit/habit-today/TodayHabitHead.tsx

```
@@ -0,0 +1,16 @@
+import Modal from "@/components/modal/Modal";
```

ESLint

[error] 'Modal' is defined but never used.

(@typescript-eslint/no-unused-vars)